src/platform/api/apiAccessService.ts

```diff
@@ -46,11 +46,6 @@ export class ApiAccessService {
             }
             return { extensionId: info.extensionId, accessAllowed: true };
         }
-        // For now, this API is only available in insiders.
```

Comment:
If we want powertoys to work, we have to allow insiders. Was thinking might just special case power toys, but we already have an allowed list so seemed redundant.